pages/testpage.tsx

```diff
@@ -1,13 +1,17 @@
-﻿import { NextPage } from 'next';
-import Layout from "../components/Layout/layout";
-import scss from "../pageStyles/index.module.scss";
+﻿import {NextPage} from 'next';
 import React from "react";
+import Layout from "../components/Layout/layout";
+import CandidateTestView from "../components/CandidateTestView/CandidateTestView";
+import {TestList} from "../components/CandidateTestView/Tests/TestList";
 
 const TestPage: NextPage = () => {
+    const testToRender = TestList[0];
     return (
         <Layout>
-            <section className={scss.content}> </section>
+            <div>Test</div>
+            <CandidateTestView number={testToRender.number} title={testToRender.title} brief={testToRender.brief} example={testToRender.example} executionTimeLimit={testToRender.executionTimeLimit} jsTips={testToRender.jsTips} expectedOutput={testToRender.expectedOutput} input={testToRender.input} inputDescription={testToRender.inputDescription} inputType={testToRender.inputType} modelAnswer={testToRender.modelAnswer} outputDescription={testToRender.outputDescription} outputType={testToRender.outputType}/>
```

Comment:
why not just pass in the `testToRender` object?

ie `<CandidateTestView test={testToRender}/>`

Reply:
Tried that first, it doesn't like it :( 
![image](https://user-images.githubusercontent.com/47330108/80392522-3054c800-88a7-11ea-89c6-9c93f62f9f6a.png)

Reply:
yep - you'll also need to change the component to accept a single prop of type `CandidateTestModel`

Reply:
It already does? 
![image](https://user-images.githubusercontent.com/47330108/80394073-419ed400-88a9-11ea-9cbc-4ed91e17c50f.png)

Reply:
not quite - right now it takes loads of props - one for each property within the `CandidateTestModel`.

If you want it to accept a prop called `test` which is a `CandidateTestModel` then that looks like

```ts
interface CandidateTestViewProps = {
    test: CandidateTestModel;
}
```

Reply:
much nicer, thank you